modules/js-and-css/optimize-embeds/load.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Module Name: Optimize Embeds
```

Comment:
So not Lazy Embeds?

Reply:
Ah, I see reasoning in description

Reply:
What do you think of "Embed Optimizer"? Not sure it's any better

Reply:
I was thinking we could expand this embed effort at some point beyond "Lazy", for example we could potentially replace YouTube embeds with a youtube-lite embed or add a facade pattern option (a placeholder you click to get the embed). 

I agree "Lazy Embeds" is more aligned with what it does now, but was thinking this module could cover all embedded content optimizations we decide to experiment with; what do you think?

Reply:
+1 I like "Embed Optimizer" and it maybe fits our other naming better, plus if we add a new icon it would be "Em" (first two letters is how we do them I believe).

Reply:
Set to "Embed Optimizer"